### DISCUSSION TITLE
Dịch trainable, non-trainable

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#108] [Terminology] [answered]
Dịch là `khả huấn luyện` và  `bất khả huấn luyện` được không nhỉ :thinking: 

cc @mlbvn/handson-ml

**Comment by ngcthuong** [comment]
Không biết bỏ từ "luyện " đi vó ổn không nhỉ 🧐?

**Reply by minhduc0711** [reply]
Em thấy hơi cụt :v

**Comment by minhduc0711** [comment, marked as answer]
Tạm chốt là `khả huấn luyện` và `bất khả huấn luyện` nhé mn
